R/mrgsim-bg.R

```diff
@@ -231,6 +232,13 @@ bg_mrgsim_d_impl <- function(data, mod, output = NULL, .seed = NULL,
       sink = output
     )
     return(output)
+  } 
+  if(.format == "parquet") { #nocov start
+    arrow::write_parquet( 
```

Comment:
drive by commentary :-) might want to consider using nanoparquet https://r-lib.github.io/nanoparquet/reference/write_parquet.html

then if possible i'd consider if can expose other options like compression/metadata/etc through `...` or otherwise, but that is prob a nice to have.

in case you hadn't seen: https://www.tidyverse.org/blog/2024/06/nanoparquet-0-3-0/

Reply:
Thanks, @dpastoor ; I did see `nanoparquet` come through. Already supporting `arrow::feather`, so makes sense to keep going with arrow for now. But I could see re-configuring this a bit to default to `parquet` and support that through `nanoparquet` and make it an easier install. 

Good idea to pass along arguments to `write_parquet()` etc. Currently `...` gets passed to `mrgsim()` but I could see control object for formatting those output files.